students/yushus/session01/break_me.py

```diff
@@ -0,0 +1,31 @@
+import sys
+
+def yushus_Exception_NameError():
+	try:
+		raise NameError	
+	except NameError:
+		print("Catched NameError")
```

Comment:
I saw your message asking to ignore `break_me.py`, but just wanted to let you know that a model solution `break_me.py` is posted at [here](https://github.com/UWPCE-PythonCert-ClassRepos/Au2018-Py210B/blob/master/solutions/Lesson01/break_me.py), so you might want to take a look. The purpose of this activity is not to use the `raise` statements to raise those exceptions, but to give some other statements that will raise those exceptions. The solution `break_me.py` does show those statements, so please check it out.

Also it looks like your indentation is more than 4 spaces, but it also looks like it's fixed in other/later source files, so just mentioning here.